ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -148,6 +151,16 @@ CSP:
   - zone_air_temperature_sensor
   - zone_air_cooling_temperature_setpoint
 
+CSPO:
+  description: "Single cooling setpoint control with occupancy."
```

Comment:
you do not have single cooling setpoint control, you have two and one is chosen based on occupancy. please adjust description.